Menu casi terminado

danielaaldaco opened the pull request:
Se avanzo a casi terminar el menu, favor de checar para añadir las demas pantallas

danielaaldaco committed:
Se termino la vista del menu para el usuario y se nombraron los botones correctamente

Yiyoxd commented:
El orden de los iconos del menu podria verse afectado segun el rol del usuario

Cambiar los iconos para que segun el orden no queden campos vacios

Yiyoxd reviewed (approved):
Buen diseño, pero tener **cuidado** con el orden de las opciones ya que podria haber problemas cuando las opciones se oculten

Yiyoxd committed:
correciones completas

danielaaldaco committed:
Merge pull request #5 from danielaaldaco/avancesMerged

correciones completas

Yiyoxd committed:
Mejoras en la funcionalidad de inicio de sesión:

- Diseño mejorado del formulario de inicio de sesión.
- Implementación de consulta SQL para autenticación de usuario.
- Manejo de errores para intentos fallidos de inicio de sesión.
- Validaciones para campos vacíos.

danielaaldaco committed:
Merge pull request #6 from danielaaldaco/avancesMerged

Mejoras en la funcionalidad de inicio de sesión:

Yiyoxd committed:
Se sjustaron detalles del formulario

Yiyoxd committed:
mensaje de permisos

danielaaldaco committed:
Merge pull request #7 from danielaaldaco/master

Merge completado

danielaaldaco committed:
Merge pull request #9 from danielaaldaco/avancesMerged

Avances merged

danielaaldaco committed:
merge de alfredo

danielaaldaco committed:
Merge branch 'Daniela-PrimeraVistaMenu' of https://github.com/danielaaldaco/ExamenTopicos into Daniela-PrimeraVistaMenu

danielaaldaco committed:
Se agregaron nuevos formularios
